src/neptune_mlflow_plugin/impl/utils.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2023, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+__all__ = [
+    "parse_neptune_kwargs_from_uri",
+    "encode_config",
+    "decode_config",
+]
+
+import base64
+import json
+import warnings
+from typing import (
+    Any,
+    Dict,
+)
+from urllib.parse import (
+    urlparse,
+    urlunsplit,
+)
+
+PLUGIN_SCHEME = "neptune"
+
+
+def encode_config(config: Dict[str, Any]) -> str:
+    """
+    Encodes configuration in the form of a dictionary into a string using base64.
+    Args:
+        config: dictionary to encode
+
+    Returns:
+        encoded configuration in the string form
+    """
+    config_str = json.dumps(config)
+
+    path = base64.b64encode(config_str.encode("utf-8")).decode("utf-8")
+
+    components = (PLUGIN_SCHEME, "", path, "", "")
```

Comment:
Can we made the URI fully correct? I mean `neptune:someBase64` doesn't looks like a proper URI. Let's try something like:
```
neptune://track/someBase64
```